src/utils.js

```diff
@@ -158,3 +158,16 @@ export function sorter ($sort) {
     return comparator;
   };
 }
+
+export function makeUrl (path, app = {}) {
+  const get = typeof app.get === 'function' ? app.get : () => {};
+  const env = get('env') || process.env.NODE_ENV;
+  const host = get('host') || process.env.HOST_NAME || 'localhost';
+  const protocol = (env === 'development' || env === 'test' || (env === undefined)) ? 'http' : 'https';
```

Comment:
This is making the assumption that SSL is being used in production, which obviously it SHOULD be but assuming it wasn't the URL would be incorrect, right?